src/LibChorus/merge/xml/generic/MergeChildrenMethod.cs

```diff
@@ -57,6 +57,16 @@ public void Run()
 			var parentNode = _ours ?? _theirs ?? _ancestor;
 			var parentStrategy = _merger.MergeStrategies.GetElementStrategy(parentNode);
 			var parentOrder = parentStrategy == null ? ChildOrder.AskChildren : parentStrategy.ChildOrderPolicy.OrderSignificance(parentNode);
+			// Determine if child order is important, we shouldn't create reordering conflicts if the order of children is unimportant
+			var childOrderMatters = parentOrder == ChildOrder.Significant;
+			if(parentOrder == ChildOrder.AskChildren && parentNode.HasChildNodes)
+			{
+				var childStrategy = _merger.MergeStrategies.GetElementStrategy(parentNode.FirstChild);
+				if(childStrategy != null && childStrategy.OrderIsRelevant)
+				{
+					childOrderMatters = true;
```

Comment:
I see. So the parent can have more than one kind of child, but they'd better all be the same for any given parent. Fair enough.